src/core/Directus/Exception/SQLException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Directus\Exception;
+
+use Directus\Exception\UnprocessableEntityException;
+
+class SQLException extends UnprocessableEntityException
+{
+    const ERROR_CODE = 404;
```

Comment:
Maybe we should skip error code 404? Feels like that'll get easily confused with "Not found"

Reply:
We can use any but I preferred the sequence from [here](https://docs.directus.io/api/errors.html#schema-error-codes)

Reply:
Yeah I agree 👍We should start using string based error codes in the next version like `ERR_UNPROCESSABLE_ENTITY` (@WoLfulus) to prevent these kind of issues as well